app.js

```diff
@@ -0,0 +1,96 @@
+// question 1
+
+// for a light switch statement you would use a if else statemet
+var light = 'bright'
+if(switchup === on){
+	return bright
+} else(swithDown === off){
+	return dark
+}
+
+//question 2
+// for a email you would use object 
+var user {
+	email: 'dare'
+	password: 'ginsing'
+
+}
+
+//question 3
+// you wourld make an variables
+var spaceship = ['hull', 'laserblaster', 'tractor beam', 'warp drive']
+
+/*question 4*/
+
+make an array 
+
+var students = ["jane", "Adam", "lisa", "Mark", "jesse", "allen"]
+
+
+//question 5
+
+would be a object
+
+var students = {
+class: wdi
+location: chicago
+studentName: ['yasmin', 'jeniffer', 'allen']
+}
+
+// question 6
+//var 
+
+var student name: 
+	{ name: "allen", location: "munster",
+	{ name: "bart", location:"chicago"
+	{ name: "jesse", location:"calfornia"
+	{ name: "caleb", location: "wisconsin"
+	{ name: "dare", location: "mexico"
+	{ name: "lisa" location: "las vegas"
+
+//take it easy
+	var rainbow = ['red', 'blue', 'green', 'yellow', 'orange', 'purple', 'indigo'];
+	console.log(rainbow.pop(blue);
```

Comment:
missing closing )
why .pop()? that's a method for removing elements from an array -- correct answer `console.log(rainbow[1])`